tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -843,6 +844,11 @@ TEST_F(ZoneUnitarySysTest, Test_UnitarySystemModel_factory)
     EXPECT_NEAR(totalAncillaryPower, thisSys->m_TotalAuxElecPower, 0.00000001);
     // at PLR very near 0.5, m_TotalAuxElecPower should be very near 75 W.
     EXPECT_NEAR(73.96002, thisSys->m_TotalAuxElecPower, 0.0001);
+    // delta temperature calculations
+    Real64 fanDT = thisSys->getFanDeltaTemp(*state, true, 1, 1.0);
+    EXPECT_NEAR(fanDT, 0.7070, 0.0001);
+    fanDT = thisSys->getFanDeltaTemp(*state, true, 0.5, 0.5);
+    EXPECT_NEAR(fanDT, 0.7070, 0.0001);
```

Comment:
Check for the new `getFanDeltaTemp` function. Here we have a constant volume fan the we expect the output to be the same regardless of the mass flow rate and air flow fraction we throw at it.